stories/Molecules/Blocks/CountryCard/CountryCard.jsx

```diff
@@ -1,22 +1,16 @@
 import React from 'react';
 import './countrycard.scss';
 import '../../../assets/scss/_grid.scss';
-import { Ctalink } from '../../../Components/UIcomponents/Buttons/Cta_link/Cta_link';
 
 export const CountryCard = ({ data }) => (
   <div className="grid-x grid-margin-x glide__slides">
     {data.map((item, index) => (
       <div className={['country-card', 'cell', 'glide__slide', `${item.size}`].join(' ')} key={index}>
         <a href={item.link} className="country-card__link">
```

Comment:
@Ashish-srij Remove this class `.country-card__link` and use parent selector `.country-card a` to style the anchor tag if required.

Reply:
@shishirphadke Done